docs/auditors/image.md

```diff
@@ -0,0 +1,41 @@
+# Image Auditor (image)
```

Comment:
Should we mention that this isn't on by default when you run `all`?

Reply:
It does get run by `all` but just only checks for images missing tags